src/components/EyesOnMe.js

```diff
@@ -1 +1,15 @@
-// Code EyesOnMe Component Here
+import React from "react"
+
+function EyesOnMe (){
+  return (
+    <div>
+      <button
+        onFocus={() => console.log("Good!")}
+        onBlur={() => console.log("Hey! Eyes on me!")}>
+        Eyes on me
+      </button>
```

Comment:
It would be awesome to move these functions in their own functions outside of the return:
```suggestion
    const handleFocus = () => console.log('Good!')
    const handleBlur = () => console.log('Hey! Eyes on me!')
    
    <button
      onFocus={handleFocus}
      onBlur={handleBlur}
    >
```